src/validators/location.js

```diff
@@ -278,7 +278,10 @@ export default class LocationValidator {
           valid = valid && this.validStreet()
           break
         case 'city':
-          valid = valid && this.validCity()
+          if (this.layout === Layouts.BIRTHPLACE)
```

Comment:
Shouldn't this be checking for `this.layout === Layouts. IDENTIFICATION_BIRTH_PLACE`?

Reply:
Yes, corrected, but I dont' think this code is called anywhere anymore